src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -50,11 +51,25 @@ public CommandResult execute() {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            Collection<String> lowerCaseKeywords = convertUpperCaseToLowerCase(keywords);
+            Collection<String> lowerCaseWordsInName = convertUpperCaseToLowerCase(wordsInName);
+            if (!Collections.disjoint(lowerCaseWordsInName, lowerCaseKeywords)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
 
+    /**
+     * Converts collection of words into lower case
+     *
+     * @param words strings to be converted
+     * @return collection of lower case words
+     */
+    private static Collection<String> convertUpperCaseToLowerCase(Collection<String> words) {
```

Comment:
What happens if there are numbers in the name/keywords?
Also, what is (String :: toLowerCase)?